documentation/modules/managing/proc-setting-broker-limits.adoc

```diff
@@ -46,6 +46,18 @@ Limits can be overridden by xref:con-configuring-client-quotas-str[user-specific
 * CPU utilization limits for each client can be set as a percentage of the network threads and I/O threads on a per-broker basis.
 * The number of concurrent partition creation and deletion operations (mutations) allowed per second can be set on a per-broker basis.
 
+.Impact on internal users
+When using the default Kafka quotas plugin, quotas are applied to all users. 
+This includes internal users such as the Topic Operator and Cruise Control, which may impact their operations.
+
+Unlike the Strimzi Quotas plugin, where internal users can be excluded using the `excludedPrincipals` option, the Kafka quotas plugin does not provide an exclusion mechanism.
+To avoid unduly limiting internal users when using the Strimzi Quotas plugin, consider tuning the quotas effectively.
```

Comment:
In this line, is it supposed to be `Kafka quotas plugin` instead of  `Strimzi Quotas plugin`?

In the previous line we mention `Strimzi Quotas plugin` has an `excludedPrincipals` option that can be used, so we could use that instead of having to tune the quotas effectively. The tuning would need to be done for  `Kafka quotas plugin` because it doesn't have an exclusion mechanism

Reply:
Yes! Fixed